### DISCUSSION TITLE
Podcast Players - any support for "Audiobook" medium yet?

### DISCUSSION THREAD

**Discussion by Dwev** [Podcastindex-org/podcast-namespace#604] [Q&A]
I can see some traction on the `music` medium, but when looking at Truefans and Fountain and Podverse, they only seem to mention `music` medium support, but nothing on `audiobook`.
I'd like to look at releasing an audiobook as a feed, but not sure how well it will be supported by apps, and the best way to promote it to listeners. 

The spec doesn't give a lot of information on what the audiobook medium could be capable of either.

So, what's the original idea behind the tag, how well will it be supported, and is anyone actually using it?

I found a grand total of 8 feeds using `audiobook` in Podcastindex, compared to 58 using `music`.

**Comment by Dwev** [comment]
Also, could it be an idea to get existing audiobook apps to support an RSS audiobook feed?

- Libby
- Chirp
- Audiobooks.com
- Everand
etc...

**Comment by samsethi** [comment]
Hi Guy 

TrueFans already supports medium=audiobooks. We just added medium=course for @bmaluijb. 

We need to understand what UI changes we need to make to tailor the UX. For example, if medium=music we change episodes > tracks and transcript to lyrics.   If medium=course we change episode > lessons.  But should we change episodes > chapters for audiobooks.  However this might confuse people with the chapters at the episode level.  Any thoughts? 

Also do you have an example feed we can test with.  Yuo said you found 8 podcasts with medium = audiobooks. 

<img width="1447" alt="Fullscreen_08_02_2024__19_06" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/de95c1b7-5459-434a-9c8b-a6acb1482f99">

**Reply by Dwev** [reply]
Yes, I just added a PR for Castopod to support the medium tag, and was looking into what would be different between the mediums for the UI, and saw it was not insignificant.

I ran into the same question about audiobook chapters - whether it would be confusing with the media file chapters.  I think I would still go ahead with "chapters", as then they can be broken down into "sections" or "sub-sections" using the chapter markers.

I also ran into an issue of what to call a podcast that isn't a podcast when we are talking about albums or audiobooks (or courses).  When I began to thinking about the difference between an audiobook podcast and a "traditional" podcast podcast, the word podcast as an umbrella suddenly seemed too narrow.  I then thought "feed" (but that felt too "engineer-ey") or "media" (but that creates confusion with the actual enclosure). 

I love the idea of courses, and think this should be added to the spec.  I'd love to know more about how this works, as that could really democratize the learning platform.  You don't have to rely on LinkedIn Learning or Udemy to publish, and don't have to pay exorbitant fees for Podia or similar - just create a "course" podcast and make it premium to monetize it.

**Comment by bmaluijb** [comment]
In my online courses, a section is often called “module”. So I would change episodes to “modules” when medium=course

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by Dwev** [reply]
Interesting - and as I mentioned in my reply to Sam, I love this idea around a "course" medium. Although, from my corporate L&D experience, I see a "module" as a collection of lessons, so I'd equate that more to the "season" level.  An episode is basically a single media file, so that makes sense as a "lesson".

I think the issue we will run into is that the player determines the UI for the listener.  So Sam might implement a course "episode" as a "module", but another player would implement it as a "lesson". The listener should be able to have freedom to choose their player, so this will mean different terms depending on the player.

The only way around this that I can think of right now, is to extend the podcast: namespace to include something like a terminology tag, that allows the podcaster to define the terms at the host level, and this is sent as part of the RSS feed to the player.

It could be a JSON like:

```
{
    "version": "1.0.0",
    "audiobook": {
        "podcast": "audiobook",
        "season": "volume",
        "episode": "chapter",
        "chapter": "sub-section"
    },
    "music": {
        "podcast": "album",
        "season": "disc",
        "episode": "track",
        "chapter": "section"
    },
    "course": {
        "podcast": "course",
        "season": "module",
        "episode": "lesson",
        "chapter": "chapter"
    }
}
```
Then you include the relevant section in the RSS feed.  Since it would be defined per feed, it could also be simplified to:
```
{
    "version": "1.0.0",
    "mediaterms": {
        "podcast": "audiobook",
        "season": "volume",
        "episode": "chapter",
        "chapter": "sub-section"
    }
}
```

Thoughts? 

_CC @samsethi @daveajones_

**Comment by Dwev** [comment]
Here is a table of some suggested terms based on the media type.

I'm not convinced that "blog" and "newsletter" belong in the podcast namespace, but if someone has a link to the discussion on why they are there that would be great.

I've included the number of feeds each medium has in the index in brackets.  Course is not official (yet).  Podcasts are the total feed minus the ones with media tags defined.

| Not Defined / Podcast (4095555) | Video (59) | Music (58) | Audiobook (8) | Film (6) | Course | _Blog (0)_ | _Newsletter (0)_ |
|---|---|---|---|---|---|---|---|
| Podcast | Vodcast (!) | Album | Audiobook | Film | Course | _Blog_ | _Newsletter_ |
| Season | Season  | Disc | Volume | N/A | Module | _Year_ | _Volume_ |
| Episode | Episode | Track | Chapter | Feature | Lesson | _Entry_ | _Issue_ |
| Chapter | Chapter | Section | Section | Section | Chapter | _Topic_ | _Header_ |

**Comment by Dwev** [comment]
Just thought I'd post this episode from the Sound School Podcast about how Pushkin is publishing audiobooks as podcasts. I wonder if Pushkin staff are aware of PC 2.0?

https://transom.org/2023/the-best-audio-storytelling-according-to-pushkin/

**Comment by samsethi** [comment]
I thought I would check Pushkin https://www.pushkin.fm/audiobooks but not one of these audiobooks is listed in the PodcastIndex?  So are they really RSS podcasts?  I am interviewing Pushkins CEO on Monday so I will ask her.  I was excited as TrueFans supports medium=audiobooks.

**Reply by Dwev** [reply]
Maybe the full audiobook is a private feed, since they charge for access. Private feeds won't appear in the index, right?

**Reply by samsethi** [reply]
We have created a 'locked value' option for podcasters in their admin dashboard. So in TrueFans you can set the minimum value you are willing to allow listeners to pay. e.g in this example §100 sats.   

In this V4V model, the listener can choose to pay that minimum locked amount or pay more but they cannot pay less.  This locked value works for music, audiobooks and courses.  The podcasters can then remove the locked value button when they feel they have accrued sufficient value for their content and then they can revert to the original V4V model where the listener pays whatever value they want, even below the suggested play value. 

We are retaining the idea of 'private feeds' for corporate podcasts and other similar content.   


<img width="1105" alt="TrueFans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/ad604e5e-9819-4d87-aec7-d0d293762983">

**Reply by Dwev** [reply]
That's a cool idea, Sam!

The interview will be on PNW right?  I look forward to it, as I'm really wanting to explore the potential for audiobooks. :)
